src/megachatapi_impl.cpp

```diff
@@ -2626,10 +2640,14 @@ void MegaChatApiImpl::closeChatRoom(MegaChatHandle chatid, MegaChatRoomListener
     if (chatroom)
     {
         chatroom->removeAppChatHandler();
-
-        removeChatRoomListener(chatid, listener);
-        removeChatRoomHandler(chatid);
     }
+    else
+    {
+        API_LOG_WARNING("Try to close nonexistent chatroom: %d", chatid);
```

Comment:
Logs should print `chatid`s in Base64 for consistency
```suggestion
        API_LOG_WARNING("Try to close nonexistent chatroom: %s", karere::Id(chatid).toString().c_str());
```